### PR TITLE
Add pslib

### PR DESCRIPTION
**Pull request by dsendkowski**

### CODE REVIEW

File: components/print/pslib/Makefile [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+CONFIGURE_OPTIONS += --disable-static
+
+
+build: $(BUILD_32_and_64)
+
+install: $(INSTALL_32_and_64)
+
+test: $(NO_TEST)
+
+REQUIRED_PACKAGES += image/library/libjpeg6
```
Comment (xen0l) [inline components/print/pslib/Makefile:51]: This seems to be an error, There should be only one libjpeg implementation as a dependency. Also the package seems to be badly linked:
<pre>
xenol@external lib (pslib) % ldd libps.so 
        libpng16.so.16 =>        /usr/lib/libpng16.so.16
        libjpeg.so.62 =>         /usr/lib/libjpeg.so.62
        libtiff.so.5 =>  /usr/lib/libtiff.so.5
        libc.so.1 =>     /lib/libc.so.1
        libz.so.1 =>     /usr/lib/libz.so.1
        libm.so.2 =>     /lib/libm.so.2
        liblzma.so.5 =>  /usr/lib/liblzma.so.5
        libjpeg.so.8 =>  /usr/lib/libjpeg8-turbo/lib/libjpeg.so.8
        libmd.so.1 =>    /lib/libmd.so.1
        libpthread.so.1 =>       /lib/libpthread.so.1
</pre>

Preferably, link to libjpeg-turbo. @pyhalov / @alarcher anything else?

File: components/print/pslib/pslib.p5m [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/include/libps/pslib-mp.h
```
Comment (xen0l) [inline components/print/pslib/pslib.p5m:25]: Are headers the same for 32-bit and 64-bit versions?

**Review by xen0l — CHANGES_REQUESTED** [review]

File: components/print/pslib/Makefile (hunk 1 above)
Comment (dsendkowski) [reply components/print/pslib/Makefile:51]: I did add these jpeg8-turbo options and 64bits library is linked correctly:
```
ldd build/prototype/i386/usr/lib/amd64/libps.so
	libpng16.so.16 =>	 /usr/lib/64/libpng16.so.16
	libjpeg.so.8 =>	 /usr/lib/libjpeg8-turbo/lib/amd64/libjpeg.so.8
	libtiff.so.5 =>	 /usr/lib/64/libtiff.so.5
	libc.so.1 =>	 /lib/64/libc.so.1
	libz.so.1 =>	 /usr/lib/64/libz.so.1
	libm.so.2 =>	 /lib/64/libm.so.2
	liblzma.so.5 =>	 /usr/lib/64/liblzma.so.5
	libpthread.so.1 =>	 /lib/64/libpthread.so.1
```
however 32bits is not:
```
ldd build/prototype/i386/usr/lib/libps.so
	libpng16.so.16 =>	 /usr/lib/libpng16.so.16
	libjpeg.so.62 =>	 /usr/lib/libjpeg.so.62
	libtiff.so.5 =>	 /usr/lib/libtiff.so.5
	libc.so.1 =>	 /lib/libc.so.1
	libz.so.1 =>	 /usr/lib/libz.so.1
	libm.so.2 =>	 /lib/libm.so.2
	liblzma.so.5 =>	 /usr/lib/liblzma.so.5
	libjpeg.so.8 =>	 /usr/lib/libjpeg8-turbo/lib/libjpeg.so.8
	libpthread.so.1 =>	 /lib/libpthread.so.1
```
Are there any additional options I should add?

File: components/print/pslib/pslib.p5m (hunk 2 above)
Comment (dsendkowski) [reply components/print/pslib/pslib.p5m:25]: Yes, they are.

File: components/print/pslib/Makefile (hunk 1 above)
Comment (pyhalov) [reply components/print/pslib/Makefile:51]: Can't say without looking at component's Makefiles

Comment (dsendkowski) [reply components/print/pslib/Makefile:51]: Ok, I think I found what causes this issue. Can I patch `configure` only or I should patch rather `configure.in`?

Comment (pyhalov) [reply components/print/pslib/Makefile:51]: Better patch configure.ac or configure.in

**Comment by dsendkowski** [comment]
I'm afraid I need to close this PR. There are too many patches required in the source code itself so that it can be used in 64bits.

**Pull request closed without merging**